service/authorization/authorization.proto

```diff
@@ -158,7 +166,13 @@ message GetDecisionsResponse {
          "id": "e2",
          "emailAddress": "alice@example.org"
        }
-     ]
+     ],
```

Comment:
I've been wondering if we need scopes on GetEntitlements. Is a scoped GetEntitlements call different than a GetDecision request with a known set of attributes and a set of entities?

Reply:
This comment should be captured in an issue instead of PR comments. Closing out.